### PR TITLE
Addition of Continue button for registration of selftest 

### PR DESCRIPTION
**Pull request by Juliehoejrisp**
Addition of continue button for registration of selftest on Android and iOS. (The UI part).

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Utils/AuthErrorUtils.cs [hunk 1]
```diff
@@ -14,36 +14,41 @@ public static class AuthErrorUtils
...
-            GoToErrorPage(parent, REGISTER_ERROR_FETCH_FHI_DATA_HEADER, REGISTER_ERROR_FETCH_FHI_DATA_DESCRIPTION, REGISTER_ERROR_DISMISS);
+            GoToErrorPage(parent, REGISTER_ERROR_FETCH_FHI_DATA_HEADER, REGISTER_ERROR_FETCH_FHI_DATA_DESCRIPTION, REGISTER_ERROR_DISMISS, REGISTER_SELFTEST_BUTTON);
             LogUtils.LogException(severity, e, errorMessage);
         }
 
-        public static void GoToErrorPage(Activity parent, string title, string description, string button, string subtitle = null)
+       
+
+
+
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.Droid/Utils/AuthErrorUtils.cs]: Delete extra empty lines

File: NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/ErrorActivities/GeneralErrorActivity.cs [hunk 2]
```diff
@@ -83,6 +84,13 @@ void Init()
             }).Run;
 
             this.Title = textFieldsBundle.GetString("title");
+
+
+            Button registerSelftestButton = FindViewById<Button>(Resource.Id.continue_button);
+            registerSelftestButton.Text = continueButtonText;
+            registerSelftestButton.ContentDescription = ErrorViewModel.REGISTER_SELFTEST_BUTTON;
+            button.ContentDescription = continueButtonText;
+
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/ErrorActivities/GeneralErrorActivity.cs:91]: Delete extra empty line

File: NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/ErrorActivities/GeneralErrorActivity.cs [hunk 3]
```diff
@@ -83,6 +84,13 @@ void Init()
             }).Run;
 
             this.Title = textFieldsBundle.GetString("title");
+
+
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/ErrorActivities/GeneralErrorActivity.cs]: Delete extra empty lines

File: NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs [hunk 4]
```diff
@@ -107,8 +107,17 @@ private void SetupStyling()
             {
                 ErrorMessageLabel.AccessibilityLabel = ErrorViewModel.REGISTER_ERROR_ACCESSIBILITY_TOOMANYTRIES_DESCRIPTION;
             }
-        }
 
+
+			StyleUtil.InitButtonStyling(ContinueWithSelftestButton, ErrorViewModel.REGISTER_SELFTEST_BUTTON);
+
+			if (ErrorTitle != ErrorViewModel.REGISTER_ERROR_NOMATCH_HEADER)
+            {
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs:114]: Fix indent

File: NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs [hunk 5]
```diff
@@ -107,8 +107,17 @@ private void SetupStyling()
...
+				ContinueWithSelftestButton.Hidden = true;
+			}
+
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs]: Delete extra empty line

**Review by pavshr — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs (hunk 4 above)
Comment (Juliehoejrisp) [reply NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs:114]: what do you mean?

Comment (pavshr) [reply NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs:114]: Allign `{ `so it is placed right under the `if`, same indentation as on the line above

Comment (Juliehoejrisp) [reply NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs:114]: yes i did that now.

Comment (pavshr) [reply NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs:114]: looks like indentation is broken in the whole file, if you have time you can fix, if not, just merge

**Review by pavshr — APPROVED** [review]

**Pull request merged**